merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/ConstraintsDSLCompilationService.java

```diff
@@ -52,8 +52,11 @@ public void close() {
   synchronized public ConstraintsDSLCompilationResult compileConstraintsDSL(final String missionModelId, final String constraintTypescript)
   throws MissionModelService.NoSuchMissionModelException
   {
-    final var missionModelGeneratedCode = JSONObject.quote(this.typescriptCodeGenerationService.generateTypescriptTypesFromMissionModel(missionModelId));
-
+    final var missionModelGeneratedCode = this.typescriptCodeGenerationService.generateTypescriptTypesFromMissionModel(missionModelId);
+    final JsonObject typescriptCode = Json.createObjectBuilder()
```

Comment:
I'd suggest renaming `typescriptCode` to something like `messageJson`, to signify that this is the payload being sent to the subprocess. (When I read `typescriptCode`, I expect that to be a string that contains the contents of a typescript file).

Reply:
I called it that because it's a JSON wrapper on generated Typescript code, but I can see what you mean. Change is done!

Reply:
Looks great, thanks